packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.css

```diff
@@ -19,8 +19,19 @@
   visibility: hidden;
 }
 
+.optionsList {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
 .option {
-  display: block;
+  padding: var(--space-small) var(--space-base);
+  cursor: pointer;
```

Comment:
we'll need to have this rule in `.option:hover` for it to have the desired effect, but we can make a note to do that in the styling PR, or we could do it now. I don't mind terribly either way!

Reply:
I've got it in my styling PR 👍